src/models/regression.py

```diff
@@ -14,12 +14,14 @@ class LinearRegression(ModelBase):
     model_name = 'linear_regression'
 
     def train(self, num_epochs: int = 1,
+              experiment: str = 'one_month_forecast',
```

Comment:
This should be a class attribute, since this information will also be needed by the `predict` function.

(This also needs to be added to the predict function)